android/anc/src/main/java/org/smartregister/fhircore/anc/data/madx/model/MadxModels.kt

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2021 Ona Systems, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.smartregister.fhircore.anc.data.madx.model
```

Comment:
what about these is specific to mADX?

Reply:
We might create a exclusive madx module but for now I created a different package for it after discussing it with @f-odhiambo

Reply:
k what I'm wondering, is there generic non-mADX functionality that these models are associated with? or are these only relevant to mADX? From the content of this class, it's not clear to me why these are mADX specific